build.sbt

```diff
@@ -89,12 +89,12 @@ lazy val `anorm-iteratee` = (project in file("iteratee"))
   .settings(
     previousArtifacts := Set.empty,
     libraryDependencies ++= Seq(
-      "com.typesafe.play" %% "play-iteratees" % "2.6.0",
+      "com.typesafe.play" %% "play-iteratees" % "2.6.1",
       "org.eu.acolyte" %% "jdbc-scala" % acolyteVersion % Test      
     ) ++ Seq(
       "specs2-core",
```

Comment:
not blocking this PR: specs dependencies are used both in several modules, so they can be declared to be reusable